Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/CarbonBlackEndpointStandard.yml

```diff
@@ -1257,25 +1255,25 @@ script:
     - contextPath: CarbonBlackDefense.Process.Results.job_id
       description: The job_id of the process search.
       type: String
-    - contextPath: CarbonBlackDefense.Process.Results.results.approximate_unaggregated
+    - contextPath: CarbonBlackDefense.Process.Results.approximate_unaggregated
```

Comment:
I see this change but I don't see a change in how you build the context in any of the commands. Are you sure the context indeed look like this?

Reply:
yes I changed it
![Screen Shot 2021-05-09 at 19 36 06](https://user-images.githubusercontent.com/78307768/117579915-e3ccdb00-b0fd-11eb-8ce4-d5418d1a39ea.png)

Reply:
in this commit
https://github.com/demisto/content/pull/11478/commits/01d7b58f9ca2fd281f0ae6b6d3b7dfc1f75d2dae